src/python/WMCore/MicroService/Service/RestApiHub.py

```diff
@@ -34,3 +34,5 @@ def __init__(self, app, config, mount):
 
         self._add({"status": Data(app, self, config, mount),
                    "info": Data(app, self, config, mount)})
+        if 'pileup' in mount:
+            self._add({"access": Data(app, self, config, mount)})
```

Comment:
I think I asked this question somewhere else. But can you please expand why the `access` endpoint is required?